assets/clp_presto/methodology.md

```
@@ -0,0 +1,55 @@
+# Presto + CLP methodology
+
+**Version:**   
+Presto Version: [presto-server-0.293](https://github.com/anlowee/presto/tree/faae543ae318f0289f5d0b537c5724e1b085a2fc)  
+Velox Version: [Velox](https://github.com/anlowee/velox/tree/5a55969d5fd21bb4bcb53645b832344ff6bbd634)  
+CLP Version: [0.4.0](https://github.com/y-scope/clp/releases/tag/v0.4.0)
+
+**File with Formatted Queries:** [Config File](/assets/clp_presto/config.yaml)
+
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)